xbmc/windowing/WinSystem.h

```diff
@@ -85,6 +85,9 @@ class CWinSystemBase
   virtual bool Restore() { return false; }
   virtual bool Hide() { return false; }
   virtual bool Show(bool raise = true) { return false; }
+  // Indicates if GUI layer is currently visible. When playing video it may be completely empty
+  // and can be disabled for performance reasons
+  virtual void SetVisible(bool visible) {}
```

Comment:
Other Embedded apps can use it. Desktop apps have the ponies and just don't care.